### ISSUE TITLE
[DOC] Please, add mode local explanation to https://docs.mitmproxy.org/stable/concepts-modes/

### ISSUE THREAD

**Issue by q2dg** [mitmproxy/mitmproxy#7344]
Maybe you could benefit from already written blog entries...
Thanks!

**Comment by mhils** [comment]
Yes! This clearly needs to happen. We plan to finish Linux support by the end of the year, and then properly add it to the docs. :)

**Comment by errorxyz** [comment]
https://docs.mitmproxy.org/stable/concepts-modes/#local-capture 🎉
